330: Remove unused fields from DB schema

coderimus opened the pull request:
### Description (*)
According to the issue description, the next fields were deleted from the database schemas and also related metadata interface methods.
1. stock_id
1. vector_type
1. preview_width
1. prview_height
1. country_name
1. thumbnail_url
1. preview_url
1. details_url
1. preminum_level_id
1. url

**IMPLEMENTATION DETAILS**

**whitelist JSON update**
The whitelist JSON for the asset table schema also updated with the native Magento command.

**Change  AssetInterface::ADOBE_IDconstant to the  AssetInterface::ID**
The  `AssetInterface::ADOBE_ID` doesn't exist. Changed it to  `AssetInterface::ID` for tests.

**Remove unused constants**
Remove ADOBE_ID constants from `CategoryInterface` and `CreatorInterface`

**Rename creatot_name to name**

### Fixed Issues (if relevant)
#330 Remove unused fields from DB schema

### Manual testing scenarios (*)
Upgrade the DB structure and try to save the asset.

coderimus committed:
330: Remove unused fields from DB schema

sivaschenko reviewed (changes requested):
@coderimus Thanks for the pull request! I think we should keep the methods in the AssetInterface as it's used to get data from the API.

coderimus committed:
Revert asset interface changes

sivaschenko reviewed (changes requested):
Thanks for the update, please update database schema for the constraint names to match the whitelist

coderimus committed:
Merge branch 'develop' into feature/remove-not-used-columns

coderimus committed:
Adjustments according review

coderimus committed:
Remove unused constants

coderimus commented:
@sivaschenko thank you for your review. Requested changes implemented. Please, check. Also, I deleted unused constants and renamed the creator_name column name to the name for the creator table.
